screengrab/spec/runner_spec.rb

```diff
@@ -36,7 +36,7 @@ def mock_adb_response(mock_response)
       end
       it 'sets custom launch_arguments' do
         expect(mock_executor).to receive(:execute)
-          .with(hash_including(command: "adb -s device_serial shell am instrument --no-window-animation -w \\\n-e testLocale en_US \\\n-e endingLocale en_US \\\n-e username hjanuschka -e build_type x500 \\\n/"))
+          .with(hash_including(command: "adb -s device_serial shell am instrument --no-window-animation -w \\\n-e testLocale en_US \\\n-e endingLocale en_US \\\n-e appendTimestamp false \\\n-e username hjanuschka -e build_type x500 \\\n/"))
```

Comment:
Hey @Hiroto-N 👋 
Thanks for making this PR 💪 

Just wondering why we are expecting `appendTimestamp false` in a command. According to the code:

https://github.com/fastlane/fastlane/blob/48ac12f577e9f4c82033adb8395442eda5dcfdad/screengrab/spec/runner_spec.rb#L31-L42

we do not set the value of `use_timestamp_suffix ` (default `true`) so we would expect getting `appendTimestamp true`, wouldn't we? 

Am I missing something? 🤔

Reply:
Ah right - When I was testing running the `fastlane screengrab` command it looked like it was picking up the correct default value even when it wasn't specified in the screengrabfile, but in the tests config[:use_timestamp_suffix] was returning nil (which resolves to false). I wasn't sure how exactly that part worked, but I'll dig a bit deeper and add tests around this flag as well.

Thanks for the feedback!

Reply:
Hey, I've updated the PR - it looks like the config is usually configured through the Configurations in `Screengrab::Options.available_options`, so in a realistic scenario its never nil (which is what was making this false before).

I've set `config[:use_timestamp_suffix] = true` for this test and added tests below that populate the config through `FastlaneCore::Configuration` and passing values there.

Hope this is ok 😃